argo_prod/solrconfig.xml

```diff
@@ -31,16 +31,86 @@
       <str name="fl"> *, score</str>
 
       <str name="qf">
-        id
-        descriptive_tiv
+        sw_display_title_tesim^5
+        contributor_text_nostem_im^3
+        topic_tesim^2
+
+        tag_ssim
+
+        originInfo_place_placeTerm_tesim
+        originInfo_publisher_tesim
+
+        content_type_ssim
+        sw_format_ssim
+        object_type_ssim
+
         descriptive_text_nostem_i
+        descriptive_tiv
         descriptive_teiv
-        object_type_ssim
+
+        collection_title_tesim
+
+        id
+        objectId_tesim
+        obj_label_tesim
+        identifier_ssim
+        identifier_tesim
+        barcode_id_ssim
+        folio_instance_hrid_ssim
+        source_id_ssim^3
+        source_id_ssi
+        source_id_text_nostem_i^3
+        previous_ils_ids_ssim
+        doi_ssim
+        contributor_orcids_ssim
       </str>
-      <str name="pf">
-        descriptive_tiv^10
+      <str name="pf">  <!-- (defType dismax, edismax: phrase boost within result set) -->
```

Comment:
made pf, pf3 and pf2 reasonable.  These are parameters that improve search results when using "dismax" or "edismax", rather than "lucene."   We currently use "dismax" per blacklight_config_helper, ... for now.